localstripe/resources.py

```diff
@@ -1527,19 +1530,30 @@ def __init__(self, url=None, limit=None):
         self.url = url
 
         self._limit = limit
+        self._starting_after = starting_after
+        self._starting_pos = 0
         self._list = []
 
     @property
     def data(self):
-        return [item._export() for item in self._list][:self._limit]
+        if self._starting_after is None:
+            return [item._export() for item in self._list][:self._limit]
+
+        for i, item in enumerate(self._list):
+            if item._export().get('id', None) == self._starting_after:
+                self._starting_pos = i + 1
+                break
+
+        return [item._export()
+                for item in self._list[self._starting_pos:]][:self._limit]
```

Comment:
The code would be more efficient with
```suggestion
        return [item._export()
                for item in self._list[self._starting_pos:self._starting_pos + self._limit]]
```